app/views/collections/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_for(collection) do |f| %>
+  <% if collection.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(collection.errors.count, "error") %> prohibited this collection from being saved:</h2>
+
+      <ul>
+      <% collection.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  
+  <div class="form-group row">
```

Comment:
It appears they have a branch for support for BS4.  That being said, now that I think about this more, we're probably going to customize these forms significantly enough that something like `bootstrap_form` might not end up having that much payoff in the end.